src/client/components/Root.js

```diff
@@ -52,20 +52,27 @@ let theme = createMuiTheme({
 
 theme = responsiveFontSizes(theme);
 
-const Root = ({ classes }) => (
-  <div className={classes.root}>
-    <MuiThemeProvider theme={theme}>
-      <I18nextProvider i18n={i18nConfig}>
-        <App />
-        <ToastContainer />
-      </I18nextProvider>
-    </MuiThemeProvider>
-  </div>
-);
+const Root = ({ classes }) => {
+  useEffect(() => {
```

Comment:
Can you comment why we need this effect?

Reply:
Should this effect run on each render? cant have an empty dependency list?